.github/workflows/superlinter.yml

```diff
@@ -0,0 +1,16 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.15
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.15
+      - name: Build with Maven
```

Comment:
Dodaj proszę cacheowanie m2 repositories żeby build trwał szybciej: https://stackoverflow.com/a/59571393

Reply:
Done, tyle ze dodalem opcje --batch-mode i --update-snapshot, z tego co czytalem to batch mode idealnie nadaje sie do testow CI a update snapshot sprawdza czy ma cos pobrac i nie pobiera jak nie musi.